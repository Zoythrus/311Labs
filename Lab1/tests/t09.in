radix
5 6
751051
179637
211035
190977
923029
