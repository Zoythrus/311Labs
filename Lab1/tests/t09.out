179637
190977
211035
751051
923029
